Buildings/Experimental/OpenBuildingControl/ASHRAE/G36/Atomic/EconEnableDisableMultiZone.mo

```diff
@@ -0,0 +1,348 @@
+within Buildings.Experimental.OpenBuildingControl.ASHRAE.G36.Atomic;
+block EconEnableDisableMultiZone
+  "Multiple zone VAV AHU economizer enable/disable switch"
+
+  parameter Boolean use_enthalpy = true
+    "Set to true to evaluate outdoor air enthalpy in addition to temperature";
+  parameter Modelica.SIunits.Temperature delTemHis=1
+    "Delta between the temperature hysteresis high and low limit";
+  parameter Modelica.SIunits.SpecificEnergy delEntHis=1000
+    "Delta between the enthalpy hysteresis high and low limits"
+    annotation(Evaluate=true, Dialog(group="Enthalpy sensor in use", enable = use_enthalpy));
+  parameter Modelica.SIunits.Time retDamFulOpeTim = 180
+    "Time period to keep RA damper fully open at disable to avoid pressure fluctuations";
+  parameter Modelica.SIunits.Time smaDisDel = 15
+    "Small time delay before closing the OA damper at disable to avoid pressure fluctuations";
+
+  CDL.Interfaces.RealInput TOut(unit="K", quantity = "ThermodynamicTemperature")
+    "Outdoor air (OA) temperature"
+    annotation (Placement(transformation(extent={{-220,250},{-180,290}}),
+        iconTransformation(extent={{-120,90},{-100,110}})));
+  CDL.Interfaces.RealInput hOut(unit="J/kg", quantity="SpecificEnergy") if use_enthalpy
+      "Outdoor air enthalpy" annotation (Placement(transformation(extent={{-220,170},{-180,210}}),
+        iconTransformation(extent={{-120,50},{-100,70}})));
+  CDL.Interfaces.RealInput TOutCut(unit="K", quantity = "ThermodynamicTemperature")
+    "OA temperature high limit cutoff. For differential dry bulb temeprature condition use return air temperature measurement"
+    annotation (Placement(transformation(extent={{-220,210},{-180,250}}),
+        iconTransformation(extent={{-120,70},{-100,90}})));
+  CDL.Interfaces.RealInput hOutCut(unit="J/kg", quantity="SpecificEnergy") if use_enthalpy
+    "OA enthalpy high limit cutoff. For differential enthalpy use return air enthalpy measurement"
+    annotation (Placement(transformation(extent={{-220,130},{-180,170}}),iconTransformation(extent={{-120,30},{-100,50}})));
+  CDL.Interfaces.RealInput uOutDamPosMin(min=0, max=1)
+    "Minimum outdoor air damper position, from EconDamperPositionLimitsMultiZone sequence"
```

Comment:
Please only use model names if you can make a hyperlink. Here, simply write in words where to get signal from. Otherwise, if we rename `EconDamperPositionLimitsMultiZone` with a conversion script, then this text will not be updated and hence references a block that no longer exists. As we have 100s of files, detecting such broken documentation is very hard, hence we want to avoid the risk of breaking it.